test/integration/scheduler_perf/util.go

```diff
@@ -118,34 +115,33 @@ func mustSetupCluster(ctx context.Context, tb testing.TB, config *config.KubeSch
 		var err error
 		config, err = newDefaultComponentConfig()
 		if err != nil {
-			tb.Fatalf("Error creating default component config: %v", err)
+			tCtx.Fatalf("Error creating default component config: %v", err)
 		}
 	}
 
-	client := clientset.NewForConfigOrDie(cfg)
-	dynClient := dynamic.NewForConfigOrDie(cfg)
+	tCtx = ktesting.WithRESTConfig(tCtx, cfg)
 
 	// Not all config options will be effective but only those mostly related with scheduler performance will
 	// be applied to start a scheduler, most of them are defined in `scheduler.schedulerOptions`.
-	_, informerFactory := util.StartScheduler(ctx, client, cfg, config, outOfTreePluginRegistry)
-	util.StartFakePVController(ctx, client, informerFactory)
-	runGC := util.CreateGCController(ctx, tb, *cfg, informerFactory)
-	runNS := util.CreateNamespaceController(ctx, tb, *cfg, informerFactory)
+	_, informerFactory := util.StartScheduler(tCtx, tCtx.Client(), cfg, config, outOfTreePluginRegistry)
```

Comment:
We have several duplicated parameters here.

Reply:
Yes, there's definitely room for future refactoring/simplification of test/integration/util to get rid of redundant parameters like this one here. I'm gradually going to do more of those, but wanted to start small(ish) initially.